app/helpers/application_helper.rb

```diff
@@ -18,6 +18,10 @@ def current_user
     end
   end
 
+  def current_locale
+    params[:locale] || I18n.default_locale
+  end
```

Comment:
putting this function in `application_helper`, which is included in `application_controller`, makes it available to all controllers and views